### ISSUE TITLE
Get Switch Rules is not properly showing rule information

### ISSUE THREAD

**Issue by rtretyak** [telstra/open-kilda#2375]
Steps:
1. Have a rule that uses OF groups installed on a switch
2. Request switch rules via NB. Save response for further comparison
3. Request switch rules via Floodlight directly
**Expected:** NB should show correct information about OF groups, close to what Floodlight shows.
**Actual:** while Floodlight responds with correct infromation:
```
  "instruction_apply_actions": {
                    "actions": "group=1"
                }
```
The NB response is lacking any information about OF group:
```
            "instructions": {
                "apply_actions": {},
                "none": null,
```

**Comment by rtretyak** [comment]
same for 'go to table' actions
fl instructions:
```
"instructions": {
                "instruction_goto_meter": {
                    "instruction_goto_meter": "32"
                },
                "instruction_apply_actions": {
                    "actions": "set_field=eth_vlan_vid->2,output=19"
                },
                "instruction_goto_table": {
                    "instruction_goto_table": "1"
                }
            }
```

Nb:
```
"instructions": {
                "apply_actions": {
                    "output": "19",
                    "set_field": {
                        "key": "vlan_vid",
                        "value": "2"
                    }
                },
                "none": null,
                "instruction_goto_meter": 32
            },
```

**Comment by niksv** [comment]
partially fixed by #2605

**Comment by andriidovhan** [comment]
new issue(#2660) was discovered while testing #2655
